eng/pipelines/maintenance-packages.yml

```diff
@@ -67,6 +67,7 @@ extends:
           enableMicrobuild: true
           enablePublishUsingPipelines: true
           publishAssetsImmediately: true
+          enableSourceBuild: true
```

Comment:
@MichaelSimons Do we require source-build be enabled for official builds?  I hadn't set this originally in my change.